### PR TITLE
CMS-1399: Add support for the soon-to-be-added 'lookup-by-path' endpoint

### PR DESCRIPTION
**Pull request by ababic**

### CODE REVIEW

File: directory_cms_client/client.py [hunk 1]
```diff
@@ -108,6 +109,26 @@ def lookup_by_full_path(
...
+        region=None,
+    ):
+        base_params = build_params(
+            fields=fields,
+            draft_token=draft_token,
+            language_code=language_code,
+            region=region,
+        )
+        return self.get(
+            url=self.endpoints['page-by-path'].format(site_id=site_id, path=path), # noqa
```
Comment (richtier) [inline directory_cms_client/client.py]: can you please do something like
```
url = self.endpoints['page-by-path'].format(site_id=site_id, path=path)
return self.get(url=url, params=base_params)
```
thereby avoiding `noqa` :)

File: directory_cms_client/client.py [hunk 2]
```diff
@@ -108,6 +109,26 @@ def lookup_by_full_path(
             },
         )
 
+    def lookup_by_path(
```
Comment (richtier) [inline directory_cms_client/client.py]: named similar to the existing `lookup_by_full_path` above. Would it be feasible to delete the existing you to update `lookup_by_full_path` handler and ask whoever is using `lookup_by_full_path` to move to `lookup_by_path`?

breaking changes are fine as long as we express those breaking changes via SemVer

Comment (kowalcj0) [reply directory_cms_client/client.py]: I don't use `lookup_by_full_path` neither `lookup_by_path` in my tests

File: directory_cms_client/client.py (hunk 1 above)
Comment (ababic) [reply directory_cms_client/client.py]: Yep, no problem :)

File: directory_cms_client/client.py (hunk 2 above)
Comment (ababic) [reply directory_cms_client/client.py]: @richtier I would feel better removing `lookup_by_path` in a separate PR, if that's okay?

Comment (ababic) [reply directory_cms_client/client.py]: Removed in 6c6bcc4

**Pull request merged**